### ISSUE TITLE
Eclipse integration

### ISSUE THREAD

**Issue by ben-manes** [melix/jmh-gradle-plugin#35]
The Eclipse Gradle plugin does not include custom sourceSets as source folders. The following snippet resolves that issue and could be performed by the jmh plugin if the eclipse plugin is detected.

```gradle
apply plugin: 'eclipse'

plugins.withType(EclipsePlugin) {
  project.eclipse.classpath.plusConfigurations += [ configurations.jmh ]
}
```

**Comment by victor2100** [comment]
Many thanks!

**Comment by taichi** [comment]
:+1:

**Comment by vyazelenko** [comment]
@aalmiray There is new PR #53 with bunch fixes ;))

**Comment by vyazelenko** [comment]
Fixed with PR #53

**Comment by ben-manes** [comment]
@vyazelenko Can you add the fix in #39 too?

**Comment by vyazelenko** [comment]
@ben-manes not sure I follow. Currently IDEA integration already treats JMH sources as test sources (see https://github.com/melix/jmh-gradle-plugin/blob/master/src/main/groovy/me/champeau/gradle/JMHPlugin.groovy#L158-L173). What is missing?

**Comment by ben-manes** [comment]
Oh okay, that's the fix in #39 that I provided and forgot the changes were integrated.

**Comment by vyazelenko** [comment]
@ben-manes :+1: :smile:
